### PR TITLE
docs - update appearance settings

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
Updates docs on appearance settings for Metabase 50.

### CODE REVIEW

**Review by albertoperdomo — APPROVED** [review]
Suggested some changes but won't block on it.

File: docs/configuring-metabase/appearance.md
Comment (jeff-bruemmer) [reply docs/configuring-metabase/appearance.md]: I decided to defer to the in-product text here:

> Hide or customize pieces of the Metabase product to tailor the experience to your brand and needs

**Pull request merged**
